certified-connectors/Connect2All On-Premises/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "constraints": {
+            "clearText": false,
+            "required": "true",
+            "tabIndex": 3
+          },
+          "description": "The Password for this api",
+          "displayName": "Password",
+          "tooltip": "Provide the Password"
+        }
+      },
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "constraints": {
+            "clearText": true,
+            "required": "true",
+            "tabIndex": 2
+          },
+          "description": "The Username for this api",
+          "displayName": "Username",
+          "tooltip": "Provide the Username"
+        }
+      }
+    },
+    "iconBrandColor": "#FA9901",
+    "publisher": "GAC Business Solutions",
+    "stackOwner":  "GAC Business Solutions"
```

Comment:
@Joerio please remove an empty whitespace here before the value.

Reply:
Removed whitespace.